lib/remote/httpserverconnection.cpp

```diff
@@ -171,6 +171,40 @@ void HttpServerConnection::ProcessMessageAsync(HttpRequest& request)
 
 	HttpResponse response(m_Stream, request);
 
+	Array::Ptr headerAllowOrigin = ApiListener::GetInstance()->GetAccessControlAllowOrigin();
```

Comment:
Better add a safety check here if we really get an ApiListener object. This prevents unwanted crashes.